Makefile

```diff
@@ -4,6 +4,7 @@ DEV_IMAGE         ?= golang:1.11
 SHORT_NAME        ?= tiller
 SHORT_NAME_RUDDER ?= rudder
 TARGETS           ?= darwin/amd64 linux/amd64 linux/386 linux/arm linux/arm64 linux/ppc64le linux/s390x windows/amd64
+LINUX_ARCH        ?= amd64 386 arm arm64 ppc64le s390x
```

Comment:
Would you mind changing this to DOCKER_ARCH_TARGETS or something equivalent so it's consistent with the naming of TARGETS?